west.yml

```diff
@@ -253,7 +253,7 @@ manifest:
       revision: e8920192b66db4f909eb9cd3f155d5245c1ae825
       path: modules/lib/uoscore-uedhoc
     - name: zcbor
-      revision: a0d6981f14d4001d6f0d608d1a427f9bc6bb6d02
+      revision: 0.6.0
```

Comment:
Should this not be a commit ID?

Reply:
We have tags now on zcbor fork. The tags are the same as in upstream.